cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_cov.cpp

```diff
@@ -79,6 +80,10 @@ static train_result call_daal_kernel(const context_cpu& ctx,
         *daal_means,
         *daal_variances);
 
+    if (!status) {
+        interop::status_to_exception(status, [](const daal::services::Status& s) { });
```

Comment:
You can provide an overload of `status_to_exception()` which don't require user to provide empty lambda.
I suggest to name it differently, e.g. `interop::status_to_exception_default()`, so user will be aware that he calls only default conversion, while in common case he has to write his own conversion.

Reply:
@rlnx